tests/test_lob.py

```diff
@@ -11,3 +11,6 @@ def test_bad_auth(self):
 
     def test_set_version(self):
         self.assertEqual(lob.api_version, 'apiVersion')
+
+    def tearDown(self):
```

Comment:
Yeah - it was causing all tests before to succeed and all the ones following to fail